tac/project-updates/2025/2025-annual-Identus.md

```diff
@@ -0,0 +1,91 @@
+
+The number of active maintainers has decreased due to restructuring the Atala Team at IOG. 
+However, the diversity of maintainers has increased over the past six months, with more companies contributing to the project.
+
+## Project Adoption
+
+Several companies are using the Identus project for prototyping and production use cases.
+
+The following companies are using the project:
+- https://socious.io/, Socious production use case.
+- https://ahau.io/, Ahau PoC.
+- https://blocktrust.dev/, - identity wallet.
+
+Developers actively participate in the weekly meetings and consider using the Identus platform for their use cases.
+
+## Goals
+
+### Performance Against Prior Goals
+
+The current project roadmap is public and can be found here: [Identus Roadmap](https://github.com/orgs/hyperledger/projects/48)  
+
+During the last six months, the project has achieved the following goals:
+- Connectionless Issuance and Verification of VC via DIDCommV2
+- DID Prism AnonCreds method to resolve the resources from the DID Prism URL (for the schema and credential definition)
+- Crypto Agility of the DID Prism Method (added Ed25519 cryptography suite)
+- Transitioning the project from the IOHK to the LF Decentralized Trust organization
+- OID4VCI specification implementation in the Cloud Agent and SDKs (for JWT VC only)
+- Major refactoring:
+-- a. improved error handling in the Cloud Agent, the Mediator, and SDKs,
+-- b. added Problem Report protocol to other protocols,
+-- c. Background jobs are replaced with the message queue (in-memory or Kafka)
+- SD-JWT-VC implementation in the Cloud Agent and SDKs
+- Status List revocation implementation in JWT VC
+- Improved documentation with the Quick Start Guide and Tutorials
+
+The roadmap is actively discussed with the community members in the GitHub [Discussions](https://github.com/hyperledger/identus/discussions)
+
+### Next Year's Goals
+
+The project has the following strategic goals for the next year:
+- Reduce maintenance cost and complexity of the Identus platform
+- Increase the Identus platform adoption by the developers and companies
+- Improve the platform interoperability with other SSI platforms
+- Improve the platform documentation and tutorials
+
+The following features are planned for the next year, among others:
+- Cardano Identus VDR
+- Bring Issuance capabilities to the SDKs
+- Continue with UAS Wrapper implementation
+- Pollux Plugin abstraction SDK
+- OIDC specifications for the SSI: OID4VCI, OID4VP, SIOP2.0
+- Authentication using the PRISM DID method
+
+### Help Required
+
+LF Decentralized Trust can help with the following:
+- Promote the Identus project within the LF Decentralized Trust community
+- Help with the workshops and hackathons
+- Help to collaborate with the LF Decentralized Trust Aries project as the Identus platform leveraging the Aries specs and protocols
```

Comment:
One suggestion is to have maintainers from your project participate in Aries meetings, and invite their maintainers' to participate in yours. Avenues for collaboration and code-sharing may open up that way.